src/oge/download_data.py

```diff
@@ -107,56 +108,77 @@ def download_pudl_data(source: str = "aws"):
     Args:
         source (str, optional): where to download pudl from, either 'aws' or 'zenodo'.
             Defaults to 'aws'.
+        build (str): whether to download the "stable" or "nightly" build
 
     Raises:
         ValueError: if `source` is neither 'aws' or 'zenodo'.
```

Comment:
You can add the `UserWarning` to the docstring.

Reply:
added